pkgs/development/tools/xcbuild/wrapper.nix

```diff
@@ -73,7 +73,7 @@ while [ $# -gt 0 ]; do
          --toolchain | -toolchain) shift ;;
          --find | -find | -f)
            shift
-           command -v $1 ;;
+           command -v $1 || exit 1 ;;
```

Comment:
This is maybe because the `xcrun` script does not have `set -e`, but when the command is not found, it'd still 'exit 0' with no output. This was a problem somewhere (swiftpm or swift-driver, not sure from memory).